Added create_vvset_snapshot() function

ansarars opened the pull request.

farhan7500 commented on Modules/hpe3par_snapshot.py:
```diff
@@ -140,7 +140,14 @@
       - "The storage system user name."
     required: true
-
+  vvset_snapshot_name:
```
Add the new action in state documentation

ansarars commented:
playbook execution:-
=================
root@CSSOSBE03-B05:/home/Arshad/Play-Book/hpe3par_ansible_module# ansible-playbook vvset_snapshot_new.yml
 [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

 [WARNING]: While constructing a mapping from /home/Arshad/Play-Book/hpe3par_ansible_module/vvset_snapshot_new.yml, line 3, column 5, found a duplicate dict key (http_proxy).
Using last defined value only.


PLAY [localhost] ****************************************************************************************************************************************************************

TASK [Gathering Facts] **********************************************************************************************************************************************************
ok: [localhost]

TASK [Create volumes] ***********************************************************************************************************************************************************
changed: [localhost] => (item=volume_ansible_a)
changed: [localhost] => (item=volume_ansible_b)
changed: [localhost] => (item=volume_ansible_c)

TASK [Create vv set] ************************************************************************************************************************************************************
changed: [localhost]

TASK [pause] ********************************************************************************************************************************************************************
[pause]
Press Enter:
ok: [localhost]

TASK [Add volumes to vv set] ****************************************************************************************************************************************************
changed: [localhost]

TASK [pause] ********************************************************************************************************************************************************************
[pause]
Press Enter:
ok: [localhost]

TASK [Create vv set snapshot] ***************************************************************************************************************************************************
changed: [localhost]

TASK [pause] ********************************************************************************************************************************************************************
[pause]
Press Enter:
ok: [localhost]

TASK [Remove volumes from Volumeset] ********************************************************************************************************************************************
changed: [localhost]

TASK [pause] ********************************************************************************************************************************************************************
[pause]
Press Enter:
ok: [localhost]

TASK [Delete Volumeset] *********************************************************************************************************************************************************
changed: [localhost]

TASK [pause] ********************************************************************************************************************************************************************
[pause]
Press Enter:
ok: [localhost]

TASK [Delete Volume "{{ volume_name }}"] ****************************************************************************************************************************************
changed: [localhost] => (item=pre-volume_ansible_a-pos)
changed: [localhost] => (item=pre-volume_ansible_b-pos)
changed: [localhost] => (item=pre-volume_ansible_c-pos)

TASK [Delete Volume "{{ volume_name }}"] ****************************************************************************************************************************************
changed: [localhost] => (item=volume_ansible_a)
changed: [localhost] => (item=volume_ansible_b)
changed: [localhost] => (item=volume_ansible_c)

PLAY RECAP **********************************************************************************************************************************************************************
localhost                  : ok=14   changed=8    unreachable=0    failed=0

root@CSSOSBE03-B05:/home/Arshad/Play-Book/hpe3par_ansible_module#
root@CSSOSBE03-B05:/home/Arshad/Play-Book/hpe3par_ansible_module#


=========================
results on 3parcli:-
=========================
Romulus cli%
Romulus cli% showvv volume_ansible*
                                                                                -Rsvd(MiB)- -(MiB)-
    Id Name             Prov Compr Dedup Type CopyOf   BsId Rd -Detailed_State-   Snp   Usr   VSize
114874 volume_ansible_a tpvv No    No    base ---    114874 RW normal             512   512    1024
114875 volume_ansible_b tpvv No    No    base ---    114875 RW normal             512   512    1024
114876 volume_ansible_c tpvv No    No    base ---    114876 RW normal             512   512    1024
---------------------------------------------------------------------------------------------------
     3 total                                                                     1536  1536    3072
Romulus cli%
Romulus cli%
Romulus cli% showvv pre*
                                                                                                   -Rsvd(MiB)- -(MiB)-
    Id Name                     Prov Compr Dedup Type  CopyOf             BsId Rd -Detailed_State-   Snp   Usr   VSize
114899 pre-volume_ansible_a-pos snp  NA    NA    vcopy volume_ansible_a 114874 RW normal              --    --    1024
114900 pre-volume_ansible_b-pos snp  NA    NA    vcopy volume_ansible_b 114875 RW normal              --    --    1024
114901 pre-volume_ansible_c-pos snp  NA    NA    vcopy volume_ansible_c 114876 RW normal              --    --    1024
----------------------------------------------------------------------------------------------------------------------
     3 total                                                                                           0     0    3072
Romulus cli%
Romulus cli%
Romulus cli%
Romulus cli% showvv pre*
no vv listed
